include/flecs/addons/cpp/mixins/entity/builder.hpp

```diff
@@ -394,6 +394,35 @@ struct entity_builder : entity_view {
         return this->override<First>(_::cpp_type<Second>::id(this->m_world));
     }
 
+    /** Mark id for auto-overriding.
+     * When an entity inherits from a base entity (using the IsA relationship)
+     * any ids marked for auto-overriding on the base will be overridden
+     * automatically by the entity.
+     *
+     * @param id The id to mark for overriding.
+     */
+    Self& override_second(flecs::id_t id) {
+        return this->add_second(ECS_OVERRIDE | id);
+    }
+
+    /** Mark component for auto-overriding.
+     * @see override(flecs::id_t id)
+     *
+     * @tparam Second The component to mark for overriding.
+     */
+    template <typename Second>
+    Self& override_second(flecs::entity_t first) {
+        return this->override(first, _::cpp_type<Second>::id(this->m_world));
+    }
+
+    /** Mark pair for auto-overriding.
+     * @param first The first element of the pair.
+     * @param second The second element of the pair.
+     */
+    Self& override_second(flecs::entity_t first, flecs::entity_t second) {
```

Comment:
I don't think this method is necessary. The `*_second` functions are necessary when the first element of a pair is an id, and the second element of a pair is a type. In this case both elements are ids, which makes this function a duplicate of:

```cpp
    Self& override(flecs::entity_t first, flecs::entity_t second) {
        return this->override(ecs_pair(first, second));
    }
```